src/api/requestsApi.tsx

```diff
@@ -426,8 +439,9 @@ const transformRequestToSP = async (
   }
 
   return {
-    // if optional Person fields have been selected, include them
+    // if Person fields have been selected, include them
     ...(orgApproverId && { orgApproverId: orgApproverId }),
+    ...(supervisorId && { supervisorId: supervisorId }),
     ...(organizationalPOC && { organizationalPOCId: organizationalPOCId }),
     ...(issueTo && { issueToId: issueToId }),
```

Comment:
Make these consistent with the above lines.

```suggestion
    ...(organizationalPOCId && { organizationalPOCId: organizationalPOCId }),
    ...(issueToId && { issueToId: issueToId }),
```